<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AssistantPageBugzillaInformation</class>
 <widget class="QWidget" name="AssistantPageBugzillaInformation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>345</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="m_explanationLabel">
     <property name="text">
      <string comment="@info/rich">Complete the bug report fields: &lt;nl /&gt;&lt;note&gt;Use English.&lt;/note&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="m_titleLabel">
     <property name="text">
      <string comment="@label:textbox">Title of the bug report</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KLineEdit" name="m_titleEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="m_detailsLabel">
     <property name="text">
      <string comment="@label:textbox">Details about the crash</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KTextEdit" name="m_detailsEdit">
     <property name="acceptRichText">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="m_distributionGroupBox">
     <property name="title">
      <string comment="@title:groupbox">Distribution method:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="KComboBox" name="m_distributionChooserCombo"/>
        </item>
        <item>
         <widget class="QCheckBox" name="m_compiledSourcesCheckBox">
          <property name="text">
           <string comment="@option:checkbox">Compiled KDE by sources</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="m_crashInfoLabel">
     <property name="text">
      <string comment="@info/rich">&lt;note&gt;The crash and system information will be automatically added to the bug report.&lt;/note&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KTextEdit</class>
   <extends>QTextEdit</extends>
   <header>ktextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
